features/Revert-To-Original-Colors/style.css

```diff
@@ -0,0 +1,333 @@
+.banner-button span {
+    color: white !important;
+}
+.os-chooser .button.active {
+    background-color: #4e97fe;
```

Comment:
Isn't the old colour `#4d97ff`?

Reply:
> Isn't the old colour `#4d97ff`?

Oh, yeah you're right. I was using a color picker instead of the console, that's my bad. It's all fixed now (thank god for find and replace! 😄 )